index.html

```diff
@@ -0,0 +1,150 @@
+                <li><a href="#join">Join</a></li>
+                <li><a href="#speakerContent">Speakers</a></li>
+                <li><a href="#footerLogo">Partners</a></li>
+                <li class="classes"><a href="#footerLogo">Classes</a></li>
+            </ul>          
+        </div>
+    </div>
+
+    <section class="firstpageCover">  
+        <nav>               
+                <div class="hamburger">
+                    <div class="line1"></div>
+                    <div class="line2"></div>
+                    <div class="line3"></div>
+                </div>                
+                <ul class="navList" id="navLi">
+                    <li class="closeBtn">&times</li>
+                    <li><a href="./about.html">About</a></li>
+                    <li><a href="#mainProgram">Program</a></li>
+                    <li><a href="#join">Join</a></li>
+                    <li><a href="#desktopSpeakers">Speakers</a></li>
+                    <li><a href="./about.html">Partners</a></li>
+                </ul>
+        </nav>
+        <section class="landingPage">
+                <div class="heading">
+                    <p>"Adorable Forex Trading"</p>
+                    <h1 class="image-Letter">FIRST YOU LEARN, THEN YOU TAKE OUT THE 'L'</h1>
+                </div>
+            
+                <div class="headingPara1">
+                    <P>Trading effectively is about assessing probabilities, 
+                        not certainties.” ― Yvan Byeajee</P>
+                </div>
+
+                <div class="headingPara2">
+                    <h4>8AM ~ 5PM MON ~ FRI</h4>
+                    <p>@ N0.31 Shopping Plaza, Old G.R.A, PH</p>
+                </div>
+        </section>
+       
+    </section>
+
+    <section>
+        <div class="mainProgram" id="mainProgram">
+            <p class="mainProgramP">Main Program</p>
+            <hr>
+            <div class="mainProDiv">
+                <div>
+                    <img src="images/account-Img.png" alt="">
+                    <h4>Investment</h4>
+                    <p>Get 15% of your investment; minimum of $500</p>
+                </div>
+                <div>
+                    <img src="images/learning-Img.png" alt="">
+                    <h4>Lecture</h4>
+                    <p>Listen to the tutors from various countries about trading</p>
+                </div>
+                <div>
+                    <img src="images/investment-Img.jpeg" alt="">
+                    <h4>Live Trading</h4>
+                    <p>Enjoy free live trading signals</p>
+                </div>
+                <div>
+                    <img src="images/account-Img.png" alt="">
+                    <h4>Account Management</h4>
+                    <p>Listen to the tutors from various countries about trading</p>
+                </div>
+                <div>
+                    <img src="images/training-Icon.png" alt="">
```

Comment:
- [ ] Please add an alt text to all your images, this will help screen readers know what image it is even if the image does not load.